enterprise_access/settings/devstack.py

```diff
@@ -81,14 +81,21 @@
 
 
 ################### Kafka Related Settings ##############################
-KAFKA_ENABLED=False
+KAFKA_ENABLED = False
 
 KAFKA_BOOTSTRAP_SERVER = 'edx.devstack.kafka:29092'
 SCHEMA_REGISTRY_URL = 'http://edx.devstack.schema-registry:8081'
-KAFKA_REPLICATION_FACTOR_PER_TOPIC=1
+KAFKA_REPLICATION_FACTOR_PER_TOPIC = 1
 
-COUPON_CODE_REQUEST_TOPIC_NAME="coupon-code-request-dev"
-LICENSE_REQUEST_TOPIC_NAME="license-request-dev"
-KAFKA_TOPICS = [COUPON_CODE_REQUEST_TOPIC_NAME, LICENSE_REQUEST_TOPIC_NAME]
+COUPON_CODE_REQUEST_TOPIC_NAME = "coupon-code-request-dev"
+LICENSE_REQUEST_TOPIC_NAME = "license-request-dev"
+ACCESS_POLICY_TOPIC_NAME = "access-policy-dev"
+KAFKA_TOPICS = [
+    COUPON_CODE_REQUEST_TOPIC_NAME,
+    LICENSE_REQUEST_TOPIC_NAME,
+
+    # Access policy events
+    ACCESS_POLICY_TOPIC_NAME,
+]
```

Comment:
question: do you think it makes sense to move some of these settings into `base.py` to act as our "sane defaults"?

Reply:
Yes, I will move these settings to `base.py`